client/src/featureform/resources.py

```diff
@@ -2278,9 +2280,27 @@ def __init__(
                 "The DatabricksCredentials requires only one credentials set ('username' and 'password' or 'host' and 'token' set.)"
             )
 
-        if not cluster_id:
+        if not self.cluster_id:
             raise Exception("Cluster_id of existing cluster must be provided")
 
+        if not self._validate_cluster_id():
+            raise ValueError(
+                "Cluster_id must follow the format: xxx-xxxxxx-xxxxxxxx, where x is alphanumeric"
+            )
+
+        if self.host and not self._validate_token():
+            raise ValueError(
+                "Token must start with 'dapi' followed by alphanumeric eg ex: dapixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-x"
```

Comment:
Similarly
`f"Invalid token: expected token in the format 'dapixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-x' but received '{self.token}'`

Reply:
Changed here - 246fc8d0d2b2949838806b0b795b04f69daba214